app/grandchallenge/workstation_configs/templates/workstation_configs/workstationconfig_detail.html

```diff
@@ -25,6 +25,10 @@ <h1>Viewer Configuration {{ object.title }}</h1>
     {% endif %}
 
     <table class="table table-hover my-3">
+           <tr>
```

Comment:
Nitpick - please tidy your templates, the PyCharm autoformatter is pretty good.

Reply:
Going to double-check my PyCharm settings, it should have picked this up.